src/arrays.jl

```diff
@@ -135,19 +138,23 @@ function Base.Broadcast.broadcasted(
     u::UniFinArr{S,V,R,P,N},
     cv::CategoricalValue) where {S,V,R,P,N}
 
-    cv in classes(u) || throw(err_missing_class(cv))
+    # we assume that we compare categorical values by their unwrapped value
```

Comment:
`cv` can be in `classes(u)` but its ref code can be different than in `classes(u)` because e.g. it could have a different order.